### ISSUE TITLE
running bin/dev results in an error: The terminfo database could not be found

### ISSUE THREAD

**Issue by esmale** [bullet-train-co/bullet_train#224]
```
rescueready@RescueReady rescue_ready % bin/dev
Removing previously linked gems.
Linking 'bullet_train-1.0.74' to '/Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bullet_train-1.0.74'.
Linking 'bullet_train-fields-1.0.15' to '/Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bullet_train-fields-1.0.15'.
Linking 'bullet_train-super_scaffolding-1.0.28' to '/Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bullet_train-super_scaffolding-1.0.28'.
Linking 'bullet_train-themes-1.0.13' to '/Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bullet_train-themes-1.0.13'.
Linking 'bullet_train-themes-light-1.0.23' to '/Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bullet_train-themes-light-1.0.23'.
Linking 'bullet_train-themes-tailwind_css-1.0.10' to '/Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bullet_train-themes-tailwind_css-1.0.10'.
system           | Tmux socket name: overmind-rescue-ready-N0yByw6io2_JxISVMMh_nq
system           | Tmux session ID: rescue-ready
system           | Listening at ./.overmind.sock
worker           | Started with pid 98338...
light-css        | Started with pid 98341...
js               | Started with pid 98339...
web              | Started with pid 98337...
light-js         | Started with pid 98340...
light-mailer-css | Started with pid 98342...
yarn run v1.22.19
$ node esbuild.config.js --watch
yarn run v1.22.19
$ esbuild `bundle exec bin/theme javascript light` --bundle --sourcemap --outdir=app/assets/builds --loader:.png=file --loader:.jpg=file --watch
yarn run v1.22.19
$ NODE_PATH=./node_modules tailwindcss -c `bundle exec bin/theme tailwind-config light` -i `bundle exec bin/theme tailwind-stylesheet light` -o ./app/assets/builds/application.light.css --postcss ./postcss.config.js --watch
js               | resolveDir /Users/rescueready/sandbox/work/sarnova/rescue_ready/app/javascript/controllers
js               | path ./**/*_controller.js
yarn run v1.22.19
$ NODE_PATH=./node_modules tailwindcss -c `bundle exec bin/theme tailwind-mailer-config light` -i `bundle exec bin/theme tailwind-stylesheet light` -o ./app/assets/builds/application.mailer.light.css --postcss ./postcss.mailer.config.js --watch
web              | /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/reline-0.3.1/lib/reline/terminfo.rb:108:in `setupterm': The terminfo database could not be found. (Reline::Terminfo::TerminfoError)
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/reline-0.3.1/lib/reline/ansi.rb:21:in `<class:ANSI>'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/reline-0.3.1/lib/reline/ansi.rb:6:in `<main>'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/reline-0.3.1/lib/reline.rb:580:in `<main>'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/3.1.0/irb/color.rb:2:in `<main>'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/debug-1.5.0/lib/debug/color.rb:4:in `<main>'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/debug-1.5.0/lib/debug/thread_client.rb:6:in `require_relative'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/debug-1.5.0/lib/debug/thread_client.rb:6:in `<main>'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/debug-1.5.0/lib/debug/session.rb:24:in `require_relative'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/debug-1.5.0/lib/debug/session.rb:24:in `<main>'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/debug-1.5.0/lib/debug.rb:3:in `require_relative'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/debug-1.5.0/lib/debug.rb:3:in `<main>'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/lib/bundler/runtime.rb:60:in `block (2 levels) in require'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/lib/bundler/runtime.rb:55:in `each'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/lib/bundler/runtime.rb:55:in `block in require'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/lib/bundler/runtime.rb:44:in `each'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/lib/bundler/runtime.rb:44:in `require'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/lib/bundler.rb:176:in `require'
web              | 	from /Users/rescueready/sandbox/work/sarnova/rescue_ready/config/application.rb:8:in `<main>'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/railties-7.0.3/lib/rails/commands/server/server_command.rb:137:in `block in perform'
web              | 	from <internal:kernel>:90:in `tap'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/railties-7.0.3/lib/rails/commands/server/server_command.rb:134:in `perform'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/railties-7.0.3/lib/rails/command/base.rb:87:in `perform'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/railties-7.0.3/lib/rails/command.rb:48:in `invoke'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/railties-7.0.3/lib/rails/commands.rb:18:in `<main>'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
web              | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
web              | 	from bin/rails:4:in `<main>'
web              | Exited
light-mailer-css | Interrupting...
worker           | Interrupting...
js               | Interrupting...
light-js         | Interrupting...
light-css        | Interrupting...
worker           | /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/activesupport-7.0.3/lib/active_support/logger_silence.rb:4:in `set_encoding': Interrupt
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/activesupport-7.0.3/lib/active_support/logger_silence.rb:4:in `require'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/activesupport-7.0.3/lib/active_support/logger_silence.rb:4:in `<top (required)>'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/activesupport-7.0.3/lib/active_support/logger.rb:3:in `require'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/activesupport-7.0.3/lib/active_support/logger.rb:3:in `<top (required)>'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/activesupport-7.0.3/lib/active_support.rb:29:in `require'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/activesupport-7.0.3/lib/active_support.rb:29:in `<top (required)>'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/railties-7.0.3/lib/rails.rb:7:in `require'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/railties-7.0.3/lib/rails.rb:7:in `<top (required)>'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/sidekiq-6.4.2/lib/sidekiq/cli.rb:268:in `require'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/sidekiq-6.4.2/lib/sidekiq/cli.rb:268:in `boot_application'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/sidekiq-6.4.2/lib/sidekiq/cli.rb:37:in `run'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/sidekiq-6.4.2/bin/sidekiq:31:in `<top (required)>'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/bin/sidekiq:25:in `load'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/bin/sidekiq:25:in `<top (required)>'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/lib/bundler/cli/exec.rb:58:in `load'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/lib/bundler/cli/exec.rb:58:in `kernel_load'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/lib/bundler/cli/exec.rb:23:in `run'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/lib/bundler/cli.rb:484:in `exec'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/lib/bundler/cli.rb:31:in `dispatch'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/lib/bundler/cli.rb:25:in `start'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/exe/bundle:48:in `block in <top (required)>'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/lib/bundler/friendly_errors.rb:103:in `with_friendly_errors'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.8/exe/bundle:36:in `<top (required)>'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/bin/bundle:25:in `load'
worker           | 	from /Users/rescueready/.rbenv/versions/3.1.2/bin/bundle:25:in `<main>'
light-css        | Exited
light-js         | Exited
worker           | Exited
light-mailer-css | Exited
js               | Exited
```

**Comment by esmale** [comment]
There was another issue that was opened with the same, or very similar, error: https://github.com/bullet-train-co/bullet_train/issues/215

The reported solution there was to update a `~/.tmux.conf` file.  I, however, have to such file on my system.  Would appreciate any help/guidance I can get.

**Comment by esmale** [comment]
Did some more research, and this appears to be a bug somewhere within the overmind/tmux/reline stack, which has been reported in a couple of places:
 * [Overmind](https://github.com/DarthSim/overmind/issues/132)
 * [Reline gem](https://github.com/ruby/reline/issues/447)

Not sure I'd call this a "fix" per se, but the way to sidestep the issue is to edit (or create) this file (`~/.config/tmux/tmux.conf`) and add the line `set -g default-terminal screen-256color` to it.  Once that is in place, I'm able to run `bin/dev` without it crashing.  

Closing the issue here since it's not a bug directly within the Bullet Train stack and there is some kind of workaround for it.

**Comment by andrewculver** [comment]
Going to reopen this because we tie into Overmind if it’s installed and it feels like this would hurt newcomers.

**Comment by esmale** [comment]
I can attest to that, as I myself am a newcomer to Bullet Train, trying to get it up and running for the first time, and this has been a problem for me.

**Comment by woahdae** [comment]
Just ran into this problem myself, workaround by @esmale worked, just FYI if you're under the impression this doesn't happen.
